apps/crn-server/test/handlers/webhooks/webhook-connect-by-code.test.ts

```diff
@@ -45,7 +45,7 @@ const user: RestUser = {
 
 describe('POST /webhook/users/connections - validations', () => {
   test('returns 400 when code is not defined', async () => {
-    const res = (await handler(
+    const res = (await unloggedHandler(
```

Comment:
Why are we using an unlogged handler? If we just don't provide a DSN it wont report. This is how we deal with it in the FE

Reply:
Unlogged handler isn't used, @peterstarling suggested leaving the 'old' handlers in place so that the tests still run and check the underlying functionality.

Reply:
So test one handler and use a different one in production?